project/Dependencies.scala

```diff
@@ -653,8 +653,8 @@ object Dependencies {
   )
 
   val `testkit-scaladsl` = libraryDependencies ++= Seq(
-    playAkkaHttpServer,
     akkaStreamTestkit,
+    playAkkaHttpServer % Test,
```

Comment:
I also test it locally. The tests in testkit itself need to have a server implementation, but on a real application the server is already added by the user. Either the default akka-http or the netty server. 

When running their own tests, users will have one or other on their classpath.